app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java

```diff
@@ -105,6 +105,21 @@ SettingsStore getInstance(final @NonNull Context aContext) {
     // The maximum size is computed so the resulting texture fits within 2560x2560.
     public final static int MAX_WINDOW_WIDTH_DEFAULT = 1200;
     public final static int MAX_WINDOW_HEIGHT_DEFAULT = 800;
+    public enum WindowSizePreset {
+        PRESET_0(800, 450),
+        PRESET_1(750, 500),
+        PRESET_2(825, 550),
+        PRESET_3(900, 600);
+
+        public final int width;
+        public final int height;
+
+        WindowSizePreset(int width, int height) {
+            this.width = width;
+            this.height = height;
+        }
+    }
+    public static int windowSizeId;
```

Comment:
I think that it is better to share information through the settings store. In this case, the caller would get the current stored width and height, and map them to one of the presets.